OEM.ASM

```diff
@@ -312,4 +326,46 @@ MAPSUP:			;Dummy function
 
 CSEG	ENDS
 	END
-

+

```

Comment:
Not too sure, just hope this does the trick.
[OEM.ASM.txt](https://github.com/dspinellis/GW-BASIC/files/4691929/OEM.ASM.txt)